### ISSUE TITLE
Schema Compare extension Source/Target database dropdowns always selecting first option

### ISSUE THREAD

**Issue by ssreerama** [microsoft/azuredatastudio#14957]
- Azure Data Studio Version:

Steps to Reproduce:

1. Open Schema compare extension and try to select source and target databases any option from the dropdown.
2. Verify the databases selected before clicking ok.
3. Verify the database names that is showing in the source and target database text fields. It is showing the different database name which is the first option.

**Comment by kisantia** [comment]
might be related to https://github.com/microsoft/azuredatastudio/issues/12779

**Comment by udeeshagautam** [comment]
@ssreerama - was the issue that its 1) showing incorrect DB when you open the dialog OR 2) that you are not able to change it. If the issue is only (1) @kisantia  please add this to you PR and we cna port it if it (2) then lets close this since its not repoing?

**Comment by ssreerama** [comment]
@udeeshagautam, This is not related to the #1. We have noticed this issue is caused because of ADS cache, when I cleared the cache on local, everything works as expected. Due to cache the schema compare dialog functionality wasn't responding with the changes and still maintaining the initial populated server/database values. I am gonna close this is Issue, as it was only reproduced in my machine and Thank you all for working with me to understand the issue.
